includes/class-sensei.php

```diff
@@ -472,6 +498,11 @@ public function initialize_global_objects() {
 		$this->Sensei_WPML = new Sensei_WPML();
 
 		$this->rest_api_internal = new Sensei_REST_API_Internal();
+
+		// Student progress factories.
+		$this->course_progress_repository_factory = new Course_Progress_Repository_Factory();
+		$this->lesson_progress_repository_factory = new Lesson_Progress_Repository_Factory();
+		$this->quiz_progress_repository_factory   = new Quiz_Progress_Repository_Factory();
```

Comment:
I've replaced the factories in 63844badf9518247a7c65b07a971076b6e97514c. I've kept the `_repository` suffix to avoid confusion with the `course_progress` model. So the code looks like this:

```php
$course_progress = Sensei()->course_progress_repository->get( $course_id, $user_id );
```